src/index.js

```diff
@@ -0,0 +1,55 @@
+import _ from 'lodash';
+import './style.css';
+import printMe from './print.js';
+const data=[
+    {
+       description:"Todo listings",
+       "completed":true,
+       "index":1
+    },
+    {
+        description:"Todo listings",
+        "completed":true,
+        "index":1
+     },
+     {
+        description:"Todo listings",
+        "completed":true,
+        "index":1
+     },
+]
+const ul=document.createElement('ul')
+const div=document.createElement('div')
+div.innerHTML=`<h3>Today's todo</h3>`
+ul.appendChild(div)
+const input=document.createElement('div')
+input.innerHTML=`<input type="text" class="enter" placeholder="Enter something" />`
+ul.appendChild(input)
+
+const addliststodom=()=>{ 
+
+data.map((item)=>{
+   const li=document.createElement('li')
+   li.innerHTML=`
+   <div class="delete">
+   <div class="flexer"><input type="checkbox" class="check"/><h5>${item.description}</h5></div>
+   <h6>del</h6>
+   </div>
```

Comment:
As requested by the previous reviewer, please make sure that your design follows the wireframe 😉:
| Wireframe  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/202858087-95ff951c-ac56-43d3-a9fa-fc8ff23fbcee.png)  | ![image](https://user-images.githubusercontent.com/42799579/202858108-4d6e6ebd-5b2c-476c-a856-6f7228c74711.png)  |

Reply:
Fixed.